stream/storage/impl/src/main/java/org/apache/bookkeeper/stream/storage/impl/cluster/ZkClusterMetadataStore.java

```diff
@@ -58,7 +58,7 @@
     private final String zkClusterAssignmentPath;
 
     private final Map<Consumer<Void>, NodeCacheListener> assignmentDataConsumers;
-    private NodeCache assignmentDataCache;
+    private org.apache.curator.framework.recipes.cache.NodeCache assignmentDataCache;
```

Comment:
Sorry I think I might be missing something. Are there overloaded types of `NodeCache` in the new version?
Why are we giving the fully qualified class name?

Reply:
NodeCache as been deprecated, there is a new API, but I wanted to keep the patch as small as possible

Reply:
I see, thanks